src/components/pages/Theme/Theme.css

```diff
@@ -6,3 +6,12 @@
   display: flex;
   gap: var(--spacing-1);
 }
+
+.dropdownBox {
+  display: flex;
+  flex-direction: column;
+}
+
+.dropdownBox .dropdown {
+  margin: 5px 0;
```

Comment:
Rather than add a new selector for the margin, we can apply a `gap` to `.dropdownBox`.